### ISSUE TITLE
Add agave font

### ISSUE THREAD

**Issue by jilen** [chrissimpkins/codeface#120]
Find an interesting font here https://github.com/agarick/agave
![screenshot from 2016-01-06 13-56-41](https://cloud.githubusercontent.com/assets/546573/12135917/592958d4-b47d-11e5-9e1d-ff35e0ed8528.png)

**Comment by chrissimpkins** [comment]
@jilen Thank you!  Mind tracking down licensing information for this font?  We will need to know whether we can redistribute it here.

**Comment by jilen** [comment]
http://an.erki.net/%7Eb/agave/license I cannot open the license page... But it seems to be something forked from MIT license. So I think it's OK to include it here.

**Comment by jilen** [comment]
There's an old link I found from arch  community https://bbs.archlinux.org/viewtopic.php?id=160106. The author states `free to use/scrutinise/modify/whatever (mit license)`

**Comment by chrissimpkins** [comment]
@jilen great! We will add it. Thanks for doing the legwork.

**Comment by chrissimpkins** [comment]
It seems that there is only a regular font.  Are you aware of an oblique or bold variant?

**Comment by jilen** [comment]
@chrissimpkins There'is an regluar font currently

**Comment by chrissimpkins** [comment]
Let's put this on hold and see if the developer moves forward with those sets.  Many syntax highlighters use oblique and bold as part of their highlighting schemes.  Will you update when/if you see these become available?

**Comment by jilen** [comment]
@chrissimpkins  Ok, I will keep track of it. Actually I have made an oblique font myself by apply skew with fontforge. https://drive.google.com/file/d/0B5wJ0iamqkclVXlZUUxzODVBNWM/view?usp=sharing

**Comment by tomByrer** [comment]
I'm personally am OK with just regular versions; only use italics for comments, & seems most highlighting themes in Atom use colors & not bold.

Could be uploaded & [flagged](https://github.com/chrissimpkins/codeface/issues/138#issuecomment-173437712) (or have missing flags or strike-though --bold-- --oblique--).

**Comment by chrissimpkins** [comment]
@jilen I am updating the entire repository as I generate new images with our new scripted image approach.  It seems many more of the fonts have one or two variants than I recalled.  I will include Agave once our scripted approach is in place.  Let's see if the images render with the single regular variant.

**Comment by chrissimpkins** [comment]
Agave is now available in the main gallery.  Thanks for the suggestion!

**Comment by agarick** [comment]
Boy am I late to the show -- I don't know why I didn't see this issue sooner.

This is so flattering. I really appreciate @jilen for opening this issue and @chrissimpkins for including my work.

Unfortunately, I don't have immediate plans to work on variants, but I'd be happy to consider any [direct suggestions for improvement or change](https://github.com/agarick/agave/issues). I also haven't really been keeping tabs on how Agave fares with linux rendering of the past few years (I actually only use Source Code Pro nowadays, heh).

But to be honest, looking at the font again (and awesome screenshot at the top, by the way!), if I do re-enter type design, I think I would lean towards a different, new kind of style.

Thanks again!

**Comment by chrissimpkins** [comment]
@agarick 👍   let us know if you make changes and we will get the fonts here updated!
